shell/platform/common/incoming_message_dispatcher.cc

```diff
@@ -17,16 +17,17 @@ void IncomingMessageDispatcher::HandleMessage(
     const FlutterDesktopMessage& message,
     const std::function<void(void)>& input_block_cb,
     const std::function<void(void)>& input_unblock_cb) {
-  std::string channel(message.channel);
+  const std::string& channel = message.channel;
```

Comment:
`message.channel` is a `const char*`, so I'm not sure what this is supposed to be changing. AFAICT this is doing the same thing, but much more mysteriously since it relies on understanding reference lifetime extension of an anonymous `std::string` instance.

Reply:
done